backend/djangoindia/bg_tasks/send_update.py

```diff
@@ -0,0 +1,24 @@
+from celery import shared_task
+from django.core.mail import send_mail
+from djangoindia.db.models.update import Update
+from django.conf import settings
+
+@shared_task
+def send_update_email(update_id):
+    update = Update.objects.get(pk=update_id)
+    for subscription in update.recipients.all():
```

Comment:
How are you making sure that update will always have an object and won't be None?

Reply:
If Update is created by admin only. so I thought that there would such case. Thanks for the review if there are any such problem. Please let me know I will make required changes. Thanks.

Reply:
might be / might not be. but from development perspective you need to handle every case